docs/sources/operators-guide/getting-started-helm-charts/_index.md

```diff
@@ -0,0 +1,298 @@
+   kubectl -n <namespace> get pod
+   ```
+
+   The results look similar to this:
+
+   ```bash
+   kubectl -n dev get pod
+   NAME                                                       READY   STATUS      RESTARTS   AGE
+   <release-name>-mimir-nginx-69fd969c-g6bkn                  1/1     Running     0          159m
+   <release-name>-minio-5757f44456-dz7xx                      1/1     Running     0          159m
+   <release-name>-mimir-distributed-make-bucket-job-f5gbj     0/1     Completed   0          159m
+   <release-name>-mimir-distributor-7db5f7c8bc-444sd          1/1     Running     0          148m
+   <release-name>-mimir-overrides-exporter-79b475c98d-sqzxx   1/1     Running     0          148m
+   <release-name>-mimir-querier-5cb58b7b9c-zwnr6              1/1     Running     0          148m
+   <release-name>-mimir-query-frontend-74d666bcd5-4dcwh       1/1     Running     0          148m
+   <release-name>-mimir-ruler-745fff6c6d-8swsv                1/1     Running     0          148m
+   <release-name>-mimir-alertmanager-0                        1/1     Running     0          148m
+   <release-name>-mimir-compactor-0                           1/1     Running     0          148m
+   <release-name>-mimir-ingester-1                            1/1     Running     0          147m
+   <release-name>-mimir-ingester-2                            1/1     Running     0          147m
+   <release-name>-mimir-store-gateway-0                       1/1     Running     0          146m
+   <release-name>-mimir-ingester-0                            1/1     Running     0          144m
+   ```
+
+1. What until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
+
+## Configure Prometheus to write to Grafana Mimir
+
+Make a choice based on whether or not you already have a Prometheus server set up:
+
+- For an existing Prometheus server:
+
+  1. Add the following YAML snippet to your Prometheus configuration file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir, based on what is defined in the existing `scrape_configs` configuration.
+
+  1. Restart the Prometheus server.
+
+- For a Prometheus server that does not exist yet:
+
+  1. Write the following configuration to a `prometheus.yml` file:
+
+     ```yaml
+     remote_write:
+       - url: http://<ingress-host>/api/v1/push
+
+     scrape_configs:
+       - job_name: prometheus
+         honor_labels: true
+         static_configs:
+           - targets: ["localhost:9090"]
+     ```
+
+     In this case, your Prometheus server writes metrics to Grafana Mimir that it scrapes from itself.
+
+  1. Start a Prometheus server by using Docker:
+
+     ```bash
+     docker run --network=host -p 9090:9090  -v <path-to>/prometheus.yml:/etc/prometheus/prometheus.yml prom/prometheus
+     ```
+
+## Configure Grafana Agent to write to Grafana Mimir
```

Comment:
I would clarify that you don't have to run both Prometheus and Grafana Agent, but you can run Grafana Agent instead of Prometheus.